src/ucp/api/ucp.h

```diff
@@ -2056,15 +2056,16 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
  *
  * @param [in]     ep       UCP endpoint that is used for the receive operation.
  * @param [in]     buffer   Pointer to the buffer to receive the data to.
- * @param [inout] count     Number of elements to receive into @a buffer as
- *                          in-parameter and number of actual elements received.
- *                          I.e. output value can be less or equal input value.
+ * @param [in]     count    Number of elements to receive into @a buffer.
  * @param [in]     datatype Datatype descriptor for the elements in the buffer.
  * @param [in]     cb       Callback function that is invoked whenever the
  *                          receive operation is completed and the data is ready
  *                          in the receive @a buffer. It is important to note
  *                          that the call-back is only invoked in a case when
  *                          the operation cannot be completed immediately.
+ * @param [out]    length   The size of the received data in bytes,
+ *                          always boundary of base datatype size. The value is
```

Comment:
@bbenton pls help to phrase it:
```
@param [out]    length   Size of the received data in bytes. Valid only if return code is UCS_OK,

@note The received data is always on the boundary on the provided @a datatype. 
The exact definition is according to the datatype.
```

Reply:
How about this for the note:
The amount of data received, in bytes, is always an integral multiple of the datatype size.